scripts/tables.R

```diff
@@ -1169,7 +1169,7 @@ tab_cost_est_prep5 <- left_join(
 ##add the priority info
 tab_cost_est_phase2 <- tab_cost_est_prep5 %>%
   filter(source %like% 'phase2') %>%
-  filter(barrier_result != 'Unknown' & barrier_result != 'Passable') %>%
```

Comment:
I don't think there is any cost estimate info in this report at the moment. The only reason I changed that line above is so that the passable crossing 6745 would show up on the interactive map. Because the `tab_cost_est_phase2` object is used in a join to populate the map objects. There are only 3 phase 2 sites for peace so I figured we might as well have all of them on the map.